Gruntfile.js

```diff
@@ -34,6 +34,12 @@ module.exports = function (grunt) {
                     timeout: 10000
                 },
                 src: ['test/top.js']
+            },
```

Comment:
Don't need any of these lines - emailClientTest is run from top.js along with all the other unit tests